.github/workflows/haskell.yml

```diff
@@ -35,6 +35,10 @@ jobs:
         cabal update
         cabal build --only-dependencies --enable-tests --enable-benchmarks
     - name: Build
-      run: cabal build --enable-tests --enable-benchmarks all
+      # We can't really keep this constraint here, since users aren't going to
+      # know they need to specify it. But we can at least have it to check that
+      # getting the flag specified is the only blocker to a successful build.
+      run: cabal build --enable-tests --enable-benchmarks all --constraint="cryptostore +use_crypton"
     - name: Run tests
-      run: cabal test test:testsuite
+      # as above
+      run: cabal test test:testsuite --constraint="cryptostore +use_crypton"
```

Comment:
Wouldn't be more simple to create a `cabal.project` like:
```
packages: .

package cryptostore
  flags: +use_crypton
```

This would not solve the problem of the configuring the `+use_crypton` flag between transitive dependencies (ie: a package using `jwt` and `crypton` will have to set the `+use_crypton` flag in its own cabal.project file), but we would avoid using command line flags in CI.